main.nf

```diff
@@ -487,7 +487,7 @@ workflow expression_reference_based {
 
     main:
         // count with featurecounts
-        featurecounts(sample_bam_ch, annotation)
+        featurecounts(sample_bam_ch, annotation, params.featurecounts_additional_params)
```

Comment:
why @MarieLataretu ? You can directly access `params.featurecounts_additional_params` from within the process (params are global) and you don't have to pass the parameter. Or do you want to be more explicit?

Reply:
I think, this makes it clearer that the process depends on this parameter!

Reply:
agree!